### PR TITLE
[[strings]] Add Catalan translation

### PR DESCRIPTION
**Pull request by jmontane**

### CODE REVIEW

**Review by biodranik — CHANGES_REQUESTED** [review]
Please edit data/countries_names.txt
The file you have edited is autogenerated from that file.

**Comment by jmontane** [comment]
Thanks! I'll try it

**Comment by jmontane** [comment]
@biodranik, I reverted commit with .json file and committed a first translation of countries_names.txt file. Translations of descriptions are missing, but I won't be working on it for the next few days. So, merge current PR, please. Thanks.

File: data/strings/strings.txt [hunk 1]
```diff
@@ -20460,7 +21010,8 @@
     tags = ios
     en = No GPS signal. Please move to an open area
     ar = لا يوجد إشارة نظام تحديد المواقع الجغرافية. يرجى الانتقال لمنطقة مفتوح
-    be = Няма сігналу GPS. Перамясціцеся на адкрытую прастору
+    be = Няма сігналу GPS. Перамясціцеся на адкрытую прасторуz
```
Comment (biodranik) [inline data/strings/strings.txt:21013]:
```suggestion
    be = Няма сігналу GPS. Перамясціцеся на адкрытую прастору
```

**Review by biodranik — APPROVED** [review]
Thank you for your contribution! I'll try to fix merge conflicts.

**Comment by biodranik** [comment]
@jmontane to finish localization, please translate data/categories.txt file. Note that it should match data/strings/types_strings.txt + add some synonyms if necessary (what users may type in the search field).

**Pull request closed without merging**